pkg/da/data_availability_header_test.go

```diff
@@ -239,3 +280,23 @@ func sortByteArrays(arr [][]byte) {
 		return bytes.Compare(arr[i], arr[j]) < 0
 	})
 }
+
+// maxDataAvailabilityHeader returns a DataAvailabilityHeader the maximum square
+// size. This should only be used for testing.
+func maxDataAvailabilityHeader() (dah DataAvailabilityHeader) {
+	shares := generateShares(appconsts.DefaultSquareSizeUpperBound * appconsts.DefaultSquareSizeUpperBound)
+	eds, err := ExtendShares(shares)
+	if err != nil {
+		panic(err)
+	}
+	dah = NewDataAvailabilityHeader(eds)
+	return dah
+}
+
+// invalidDah returns a DataAvailabilityHeader with an odd number of row roots.
+// This should only be used for testing.
+func invalidDah() (dah DataAvailabilityHeader) {
+	dah = MinDataAvailabilityHeader()
+	dah.RowRoots = append(dah.RowRoots, bytes.Repeat([]byte{1}, 32))
```

Comment:
[Suggestion] Should we consider replacing direct access to individual fields of a `DataAvailabilityHeader`, such as `dah.RowRoots`, with a getter method to reduce unnecessary flexibility?

Reply:
Deleted this function in https://github.com/celestiaorg/celestia-app/pull/2095/commits/8ef4bd6158cf8338f92a44f87e57a2cfcd80254b